Controllers/ServerEventsController.cs

```diff
@@ -61,8 +60,9 @@ public async Task SSE() {
             }
 
 			// Keep the server alive
-			// This feels very dodgy
-            while (true) {
+			// This still feels very dodgy
+			
+            while (cancellationToken == default) {
```

Comment:
Hmm, comparing `cancellationToken` to `default` seems dodgy... Any special reasons? Why not use `cancellationToken.IsCancellationRequested`? It does work, though!

Reply:
I also tried using `while (!cancellationToken.IsCancellationRequested)` but it didn't work at all. I really just went by whether it worked or not.

Reply:
Turns out that infinite loop wasn't necessary at all! I've removed it, as well as the `CancellationToken` shenanigans, and it seems to be working fine.